pyproprop/processed_property.py

```diff
@@ -7,6 +7,8 @@
 
 """
 
+from numbers import Real
+import numpy as np
```

Comment:
Numpy is third party not standard library, import should be separated below

Reply:
Addressed in commit 5f733d9cdce2f00a55fa62ce381a13f017a3a960